### ISSUE TITLE
GPX File

### ISSUE THREAD

**Issue by Jonathanwb** [kahopoon/Pokemon-Go-Controller#25]
I'm not quite sure what needs to be in the GPX file, are we supposed to put 

"{"lng":"114.132530212402","lat":"22.3636264801025"}" This inside, or make a new .gpx somehow?

**Comment by brendanvu1** [comment]
no. you can either use a gpx file for directions from point A to point B, or for one specific location I believe. I'm not an expert on this, but I used http://gpx-poi.com It creates a gpx.txt file for you based on the information you give it for the location. The "file name" is what you put in that space. So I renamed mine "PokemonLocation.gpx" and that's what I use, which is basically my home address. Remember to remove the .txt in the file name so it converts to a gpx file format.

**Comment by Jonathanwb** [comment]
Yeah but the Pokemon Controller isnt editing that file, for whatever reason for me

**Comment by brendanvu1** [comment]
Oh yeah same for me. I posted something earlier today in regards to that in one of the other issue forums

**Comment by brendanvu1** [comment]
I honestly don't know how to proceed from where I'm at

**Comment by g3crossfire** [comment]
so how do we fix this? im stuck on this part

**Comment by devmustache** [comment]
Im stuck too

**Comment by ldminh94** [comment]
Put your controller's ip in the "readAndChangeXML" file, then run this file in terminal. If you see the terminal updates the lng & lat, you should be good. GPX file will update itself.

**Comment by g3crossfire** [comment]
When i run "readAndChangeXML" i get this....

MacBook-Pro:Pokemon-Go-Controller gavinsaunders$ ./readAndChangeXML.py
./readAndChangeXML.py: line 1: import: command not found
./readAndChangeXML.py: line 2: import: command not found
./readAndChangeXML.py: line 3: import: command not found
./readAndChangeXML.py: line 5: lastLat: command not found
./readAndChangeXML.py: line 6: lastLng: command not found
./readAndChangeXML.py: line 8: syntax error near unexpected token `('
./readAndChangeXML.py: line 8:`def getPokemonLocation():'

**Comment by EmDee** [comment]
@g3crossfire It's a python script, thus, you need to run it with python: `python readAndChangeXML.py`

**Comment by JensDebergh** [comment]
1. You have to install `PokemonController` onto a device. 
2. Connect it using wifi
3. Get the ip address of the  device running `PokemonController` Wifi > Info > Ip address
4. Edit `readAndChangeXML.py` to the ip address you scribbled down
5. Do the same of `autoclick.py`
6. Run `readAndChangeXML.py`, next run `autoclick.py`
